package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "express-typescript-starter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "build": "tsup src/*",
+    "watch": "tsup src/* --watch",
```

Comment:
why are we pre-opting people into `tsup` as a tool here? pretty sure you can build just with `tsc` and run with `ts-node` or similar?

Reply:
hmm I guess we can do that yea

Reply:
the main typescript compiler that ships with typescript is deprecated?

Reply:
https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
https://www.npmjs.com/package/tsc shows deprecated on npm

Reply:
`tsc` is a command built into `typescript`, the `tsc` package is something completely separate

Reply:
ahh gotcha